### ISSUE TITLE
Statistics for meals

### ISSUE THREAD

**Issue by drunken-dwarf** [Faltenreich/Diaguard#24]
I'd like to view daily/weekly/monthly intake of calories, sugar, carbohydrates, etc.
The data is present, it is just not shown in the statistics page.

**Comment by Faltenreich** [comment]
It should be shown in the statistics page if you toggle the category to "Meal" by clicking on the top-most Spinner

![Screenshot_1646945127](https://user-images.githubusercontent.com/7239950/157751438-16199e6d-3586-47d1-a2a5-ff9dffaf8fe4.png)
:

**Comment by drunken-dwarf** [comment]
Unless I have missed it entirely, this screen only shows my carbohydrates/BEs/KEs, not any of the other values like calories, sugar, fat, fibers, etc.

On top of that a "today" or custom date selector for the second dropdown would also be nice to get accurate values of just a single day.

**Comment by Faltenreich** [comment]
You are right, this screen concentrates on carbohydrates as they are the most interesting value for Diabetics. I agree that other values may be of interest of well but this would require some adjustment of entering meals. Currently calories, fat, fibers and other values are only tracked when entering a meal with the support of the food database.

I agree that "Today" and a date selector would be beneficial for the statistics.
